mvrx/src/main/kotlin/com/airbnb/mvrx/BaseMvRxFragment.kt

```diff
@@ -12,30 +12,25 @@ import androidx.lifecycle.LifecycleOwner
  */
 abstract class BaseMvRxFragment(@LayoutRes contentLayoutId: Int = 0) : Fragment(contentLayoutId), MvRxView {
 
-    private val mvrxViewIdProperty = MvRxViewId()
-    final override val mvrxViewId: String by mvrxViewIdProperty
+    private lateinit var viewDelegate: DelegateMvRxView
+
+    final override val mvrxViewId: String get() = viewDelegate.mvrxViewId
 
     override fun onCreate(savedInstanceState: Bundle?) {
-        mvrxViewIdProperty.restoreFrom(savedInstanceState)
         super.onCreate(savedInstanceState)
+        viewDelegate = DelegateMvRxView(
```

Comment:
might be nice to provide a fragment specific implementation of this to make it easier to drop in for the common case

Reply:
Sounds good, will add an extension for it.